Dockerfile

```diff
@@ -13,16 +13,14 @@ RUN \
 RUN   dnf install -y python36 python3-pip
 RUN   alternatives --set python /usr/bin/python3
 RUN   alternatives --install /usr/bin/pip pip /usr/bin/pip3 1
-RUN		pip install argparse requests psycopg2-binary==2.8.6
-RUN   dnf install -y initscripts
+RUN		pip install argparse requests
+RUN   dnf install -y initscripts python3-psycopg2
```

Comment:
psycopg2-binary wasn't working for me because they don't provide arm64 binaries yet, so instead installing by yum repo which works.